app/scripts/lib/transaction/eip5792.ts

```diff
@@ -140,9 +194,6 @@ export async function processSendCalls(
     paramFrom ??
     (messenger.call('AccountsController:getSelectedAccount').address as Hex);
 
-  const dismissSmartAccountSuggestionEnabled =
-    getDismissSmartAccountSuggestionEnabled();
-
   const batchSupport = await isAtomicBatchSupported({
```

Comment:
The above `from`, and this request, and the `keyringType` below are all specific to batch so could they go in `processMultipleTransaction` to avoid the unnecessary network requests and minimise the logic?

Reply:
PR is updated